presto-server/src/main/java/com/facebook/presto/Dummy.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto;
+
+public class Dummy
```

Comment:
Please add a comment for why this class is needed.

Reply:
Will add a comment. This is needed to generate the jar. This is enforced by nexus and the `takari-lifecycle-plugin` was generating this, where the default maven package plugin skips if there are no class files. So adding this forces the package to be generated.